### DISCUSSION TITLE
Welcome to react-inner-image-zoom Discussions!

### DISCUSSION THREAD

**Discussion by laurenashpole** [laurenashpole/react-inner-image-zoom#44] [General]
## 👋 Welcome!

Since not everything is an issue, I thought I'd try out Discussions in this repo as a place to connect. Feel free to:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members. (Please be welcoming others and open-minded. Remember that this is a community we build together 💪.)

I'll kick things off with a couple questions I've been wondering about lately:

1. With the React hooks upgrade, the component's `ref` isn't automatically available. Was anyone using it? Would it be useful to forward it?
2. Would there be any interest in a Next.js specific version using [next/image](https://nextjs.org/docs/api-reference/next/image) instead of the standard `img` tag? 

Thanks for reading!

**Comment by a4vg** [comment]
I'd be interested in a react-inner-image-zoom compatible with next/image.

**Comment by jkhart47** [comment]
I would also be interested in a react-inner-image-zoom compatible with next/image

**Comment by abdulbasit-94** [comment]
Yes that's awesome, Please do that.
